prow/config.yaml

```diff
@@ -13738,6 +13738,33 @@ periodics:
       secret:
         secretName: velodrome-influxdb
 
+- name: periodic-features-no-freeze
+  interval: 1h
+  agent: kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:v20170808-abf66782
+      args:
+      - |-
+        --query=repo:kubernetes/features
+        label:lifecycle/frozen
+      - --token=/etc/token/bot-github-token
+      - |-
+        --comment=Feature issues opened in `kubernetes/features` should never be marked as frozen.
+        Feature Owners can ensure that features never become rotten by consistently updating their states across release cycles.
+
+        /remove-lifecycle frozen
+      - --template
```

Comment:
I don't actually think you're using any template functionality (contrast with https://github.com/justaugustus/test-infra/blob/332e999331ca7259e4b21598eb1a24ca938e1b70/prow/config.yaml#L14063)